### PR TITLE
Changed `EndaqCloud.get_file()` to use `endaq.ide.get_doc()`

### PR DESCRIPTION
**Pull request by StokesMIDE**
Fix for #162. 

Note that this is not yet tested. This will remain draft until it's been checked.

### CODE REVIEW

**Comment by StokesMIDE** [comment]
I think I was keeping this a draft because there were no tests. However, there are currently no tests for *any* cloud component. I'd like to merge this now, and we can do a full set of tests for the submodule in a future PR.

@shanlyMIDE @SamRagusa Please review. I will merge after approval.

**Review by shanlyMIDE — APPROVED** [review]
looks good!

**Pull request merged**
